src/parser/paladin/holy/modules/Buffs.js

```diff
@@ -48,6 +48,10 @@ class Buffs extends CoreBuffs {
         enabled: combatant.hasTalent(SPELLS.HOLY_AVENGER_TALENT),
         timelineHightlight: true,
       },
+      {
+        spellId: SPELLS.AVENGING_WRATH.id,
```

Comment:
This spell, as well as AC, already exist in the buff file. Shouldn't those be updated instead?

Reply:
Yea, you are correct.  This is fixed in the latest PR.